dot-studio/hab_toolchain

```diff
@@ -36,42 +36,6 @@ document "sup-log" <<DOC
   Tail the Habtiat Supervisor's output (alias 'sl').
 DOC
 
-# The following are additional helpers commands we provide.
-document "enforce_hab_version" <<DOC
-  Ensure that the installed version of the hab toolchain is >= the given version.
-
-  @(arg:1) The required version of Habitat
-DOC
-function enforce_hab_version() {
```

Comment:
Some studios have a dependency for this to exist, I think we should try to have a pattern of, instead of deleting functions, deprecating them with a nice error message like:
```bash
function enforce_hab_version() {
  echo "DEPRECATED: (enforce_hab_version) Please explain what the change was and what do users need to do"
  return 0
}
```